config/initializers/session_store.rb

```diff
@@ -1,9 +1,8 @@
 # Be sure to restart your server when you modify this file.
 
-Rails.application.config.session_store :cache_store, key: '_cryptopus_session', expire_after: 5.minutes
+Rails.application.config.session_store :cache_store, key: '_cryptopus_session', expire_after: 5.minutes, same_site: :lax
```

Comment:
same_site: :strict doesn't work, does it ?

Reply:
It truly does work 😅 yesterday, it didn't